syft/frameworks/torch/differential_privacy/pate.py

```diff
@@ -0,0 +1,286 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Does this copyright belong here?

Reply:
I *think* so - when you include just one file written (primarily) by other authors I think you just throw the copyright on the top of the file.